server/apiv2/handlers/members.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pingcap/kvproto/pkg/pdpb"
+	"github.com/pingcap/log"
+	"github.com/tikv/pd/pkg/errs"
+	"github.com/tikv/pd/pkg/etcdutil"
+	"github.com/tikv/pd/server"
+	"go.uber.org/zap"
+)
+
+// GetMembers returns the PD members.
+func GetMembers() gin.HandlerFunc {
+	return func(c *gin.Context) {
```

Comment:
Could we define `GetMembers` as a gin.HandlerFunc directly such as the following?
```
func GetMembers(c *gin.Context) {
	svr := c.MustGet("server").(*server.Server)
	members, err := getAllMembers(svr)
	if err != nil {
		c.AbortWithStatusJSON(http.StatusInternalServerError, err.Error())
		return
	}
	c.IndentedJSON(http.StatusOK, members)
}
```
This looks better for me aha 😄